### ISSUE TITLE
Console outputs many messages when DPE is enabled

### ISSUE THREAD

**Issue by forkercat** [o3de/o3de#14867]
**Describe the bug**

Prerequisite: `ed_enableDPE` is on

When DPE is enabled in editor and Entity Inspector is opened, there is a lot of warning messages showing in console.

*Warning Type 1:*

```
[Warning] (JSON Serialization) - Unable to retrieve description for type {2C656E00-12B0-476E-9225-5835B92209CC}.
    Reason: The operation has halted during a retrieve info operation for '<root>' because information was missing.
```

*Warning Type 2:*

```
[Warning] (Document Property Editor) - got patch for unimplemented PropertyHandler
```

*Warning Type 3:*

Related to DPE registry settings. But it can't be consistently reproduced. I'll add details later.

**Steps to reproduce**
1. Enable DPE flag
1. Create an empty level
1. Create an entity and see it in Entity Inspector
1. Check the console

**Expected behavior**

No warning messages

**Actual behavior**

<img width=800 src="https://user-images.githubusercontent.com/11157226/222005846-197fd870-f93c-4802-906e-ec144bc49793.png">

**Screenshots/Video**

See screenshots above

**Found in Branch**

Development

**Desktop/Device**
* Device: PC
* OS: Windows 10
* CPU: AMD Ryzen 5950x
* GPU: NVIDIA RTX 3070
* Memory: 64 GB

**Additional context**

None

**Comment by forkercat** [comment]
https://github.com/o3de/o3de/pull/15431 can fix most warnings.

Looks like the remaining warnings are:
- DPE setreg file is missing
- Got patch for unimplemented PropertyHandler (when deleting a component)

**Comment by michabr** [comment]
Closing this ticket, and creating new ticket for one remaining warning message https://github.com/o3de/o3de/issues/15924.
